### DISCUSSION TITLE
How to anonymize the object for WADO ?

### DISCUSSION THREAD

**Discussion by luxunxiansheng** [dcm4che/dcm4chee-arc-light#2973] [Q&A] [answered]
The doc gives the method to de-indentify for SCU/SCP,   is there a quick way to do this for wado ?

**Comment by luxunxiansheng** [comment]
I mean for wado it is possible to config specific tags as scp/scu does

**Comment by gunterze** [comment, marked as answer]
WADO applies configure [Archive Attribute Coercions](https://dcm4chee-arc-cs.readthedocs.io/en/latest/networking/config/archiveAttributeCoercion.html) with Attributes 

Name | Value
-- | --
DIMSE | C_STORE_RQ
DICOM Transfer Role | SCP

**Reply by luxunxiansheng** [reply]
Thanks a lot. It works.
